libsrc/nes/tgi/nes-64-56-2.s

```diff
@@ -242,6 +225,7 @@ SETCOLOR:
 ;
 
 SETVIEWPAGE:
+;       rts
```

Comment:
So far this type of optimization was from my perspective deliberately not done in drivers because the functions are access via a jump table. I can however follow you that it is appropriate to apply the optimization to drivers too. However I'd appreciate if it would be applied to all drivers for the sake of consistency.

Reply:
Is your thumb-up to be interpreted as "Yes, I'll check out the other drivers too and remove those RTSes." ?

Reply:
Maybe a misunderstanding, I just want to mark the fall-through. It's not an incomplete implementation like outtext.

Reply:
No, just this one time no misunderstanding ;-) This is totally unrelated to OUTTEXT. I'm simply trying to motivate you to apply the same fall-through to all cc65 drivers. I'm pretty sure that there are quite some places...

Reply:
I see...